libraries-3/src/main/java/com/baeldung/airline/CommandLine.java

```diff
@@ -0,0 +1,17 @@
+package com.baeldung.airline;
+
+import com.github.rvesse.airline.annotations.Cli;
+import com.github.rvesse.airline.help.Help;
+
+@Cli(name = "baeldung-cli", 
+description = "Baeldung Airline Tutorial",
+defaultCommand = Help.class, 
+commands = { DatabaseSetupCommand.class, LoggingCommand.class, Help.class })
+public class CommandLine {
+	 
```

Comment:
Please use the standard formatter on all the files and correct the indentation. You're using a combination of tabs and spaces.

Reply:
Fixed